introduction/index.md

```diff
@@ -3,13 +3,11 @@
 
 This document introduces the concept of a Common Specification for Information Packages. It aims to serve three main purposes:
 
-1.	Establish a common understanding of the requirements which need to be met in order to achieve interoperability of Information Packages;
+1.	Establish a common understanding about the requirements which need to be met in order to achieve interoperability between repositories and other information systems with respect to the use of Information Packages;
```

Comment:
This changes the focus of the introduction from IP interoperability to system interoperability. Isn't it too limiting?

Reply:
The idea behind the proposed change is that "interoperability" it is actually about (repository) systems being able to handle the information packages. 
In this sense, interoperability is not a characteristic of the information package, but the use of E-ARK compliant information packages is a means to achieve system interoperability.
Regarding the CS defining a common base for SIP, AIP, and DIP the purpose is also to conversions make conversions  - from SIP to AIP or from AIP(s) to DIP - more efficient. Here it is about interoperability between the systems or modules for SIP creation, storage, or access.